src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -3130,6 +3142,91 @@ private TdsOperationStatus TryProcessEnvChange(int tokenLength, TdsParserStateOb
             return TdsOperationStatus.Done;
         }
 
+        /// <summary>
+        /// Reads an enhanced routing token from the stream and populates the provided <see cref="SqlEnvChange"/> object.
```

Comment:
Not required, but I think it'd be really nice to have a layout of the token structure in a comment either a remarks or comment in the body, maybe?